javascriptv3/example_code/personalize/src/personalize_createDataset.js

```diff
@@ -0,0 +1,50 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+ABOUT THIS NODE.JS EXAMPLE: This example works with the AWS SDK for JavaScript version 3 (v3),
+which is available at https://github.com/aws/aws-sdk-js-v3.
+
+Purpose:
+createDataset.js demonstrates how to create a dataset with Amazon Personalize. 
+A dataset is a container for data that you upload to Amazon Personalize.
+See https://docs.aws.amazon.com/personalize/latest/dg/API_CreateDataset.html.
```

Comment:
For more information, see https://docs.aws.amazon.com/personalize/latest/dg/API_CreateDataset.html.